### PR TITLE
Update the build for the atomic container

### PR DESCRIPTION
**Pull request by stefwalter**
Make it work with the commands like 'atomic run' and 'atomic install', along with numerous other tweaks to get this working.

### CODE REVIEW

File: tools/atomic-install [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/sh -eu
+
+# This is the install script for Cockpit when run in a privileged
```
Comment (mvollmer) [inline tools/atomic-install]: ...container.

File: tools/atomic-run [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -eu
+
+# This is the startup script for Cockpit when run in a privileged
```
Comment (mvollmer) [inline tools/atomic-run]: ...container.

File: tools/atomic-run [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# We copy cockpit-bridge and related stuff into the host, so that
+# they work correctly there.
```
Comment (mvollmer) [inline tools/atomic-run]: This isn't true anymore, right?

File: tools/atomic-uninstall [hunk 4]
```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -eu
+
+# This is the install script for Cockpit when run in a privileged
```
Comment (mvollmer) [inline tools/atomic-uninstall]: "uninstall script"  "...run in a privileged container".

**Comment by stefwalter** [comment]
Fixed mistakes in comments. Added polkit dependency.

**Comment by stefwalter** [comment]
Rebased.

**Pull request closed without merging**